fe/src/main/java/org/apache/doris/catalog/Partition.java

```diff
@@ -344,6 +363,9 @@ public void write(DataOutput out) throws IOException {
         }
 
         out.writeLong(visibleVersion);
+        if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_88) {
+            out.writeLong(visibleVersionTime);
```

Comment:
If you do not persist the visibleVersionTime in edit log, than write it here is meaningless.